portal-impl/src/portal.properties

```diff
@@ -1395,8 +1395,10 @@
     # Set the max number of parameters for an IN clause.
     #
     # Env: LIFERAY_DATABASE_PERIOD_IN_PERIOD_MAX_PERIOD_PARAMETERS_OPENBRACKET_ORACLE_CLOSEBRACKET_
+    # Env: LIFERAY_DATABASE_PERIOD_IN_PERIOD_MAX_PERIOD_PARAMETERS_OPENBRACKET_SQLSERVER_CLOSEBRACKET_
     #
     database.in.max.parameters[oracle]=1000
+    database.in.max.parameters[sqlserver]=2000
```

Comment:
@marianoalvarosaiz, I think we should create one new property, such as **database.max.parameters[sqlserver]**. Because the current property if for **IN clause**. In SQLSERVER, it requires the SQL statement does not allow more than 2100 parameters. This is different.